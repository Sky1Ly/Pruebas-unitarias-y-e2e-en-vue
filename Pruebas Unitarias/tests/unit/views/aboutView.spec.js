import { shallowMount } from '@vue/test-utils'

import AboutView from '@/views/AboutView.vue'

describe('AboutView', () => {
    test('Validación de match con el snapshot', () => {
        //Selección del componente de prueba
        const wrapper = shallowMount(AboutView)


        //Asersion
        expect(wrapper.html()).toMatchSnapshot()
    })
})