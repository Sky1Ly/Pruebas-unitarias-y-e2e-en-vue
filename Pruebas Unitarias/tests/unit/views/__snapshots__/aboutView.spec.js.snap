// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView Validaci√≥n de match con el snapshot 1`] = `
<div>
  <h1>AboutView</h1>
</div>
`;
